src/base/projects/PeerSelector.spec.ts

```diff
@@ -57,39 +56,9 @@ describe('Logic', function () {
 describe("Layout", () => {
   it("should highlight the current peer", () => {
     render(PeerSelector, {
-      props: { ...defaultProps, peersDropdown: true }
+      props: { ...defaultProps }
     });
+    cy.get("div.selector").click();
     cy.get("div.dropdown-item").should("have.class", "selected");
   });
 });
-
```

Comment:
Couldn't get the tests to run locally to debug this: https://github.com/radicle-dev/radicle-interface/issues/318.